deployment/docker/README.md

```diff
@@ -0,0 +1,39 @@
+# OODT-docker
+
+Automatic RADIX build for Apache OODT
+
+## Usage
+
+You should edit docker file replacing the parameters below with whatever you want before build the image.
+
+1. The groupId is a place to specify your company's namespace. 
+    ```dockerfile
+    -DgroupId = com.mycompany
+    ```
+2. The artifactId is a place to specify a short name of your project. 
+    ```dockerfile
+    -DartifactId = oodt
+    ```
+3. The version indicates the initial version label for your project. 
+    ```dockerfile
+    -Dversion = 0.1
+    ``` 
+4. The oodt flag indicates the version of OODT that you want your project to be built on. N.B., this should most likely match [the most recent version of OODT](https://search.maven.org/search?q=g:org.apache.oodt).
+    ```dockerfile
+    -Doodt = 1.2.5
+    ```
+
+### Building OODT Docker Image
+Execute below command in a prompt which is opened where the Docker file placed. Replace any thing with `oodt_docker` as it is the name of your docker image.
```

Comment:
```suggestion
Execute below command in a prompt which is opened where the Docker file placed. Replace  `oodt_docker` with any name as it will be the name of your docker image.
```

Reply:
Fixed as suggested